src/ocrd/mets_server.py

```diff
@@ -438,15 +438,18 @@ def kill_process(mets_server_pid: int):
             pass
 
     def shutdown(self):
+        # os._exit because uvicorn catches SystemExit raised by sys.exit
+        # _exit(0)
+        pid = os.getpid()
+        self.log.info(f"Shutdown method of mets server[{pid}] invoked, sending SIGTERM signal.")
+        os.kill(pid, signal.SIGTERM)
         if self.is_uds:
             if Path(self.url).exists():
                 self.log.warning(f"Due to a server shutdown, removing the existing UDS socket file: {self.url}")
                 Path(self.url).unlink()
-        # os._exit because uvicorn catches SystemExit raised by sys.exit
-        _exit(0)
 
     def startup(self):
-        self.log.info("Starting up METS server")
+        self.log.info(f"Configuring up the Mets Server")
```

Comment:
```suggestion
        self.log.info(f"Configuring the Mets Server")
```

Though I found `Starting up` clearer.

Reply:
Yes, but then there is another log right before starting the uvicorn server at the end of the method. I thought a differentiation between the two may be good.